Formula/guile.rb

```diff
@@ -48,6 +49,15 @@ class Guile < Formula
     cause "Segfaults during compilation"
   end
 
+  # http://debbugs.gnu.org/cgi/bugreport.cgi?bug=23870
+  # https://github.com/Homebrew/homebrew-core/issues/1957#issuecomment-229347476
+  if MacOS.version >= :el_capitan
```

Comment:
Not completely sure, to be honest. Wasn't sure if impact would stretch to anything using Xcode 8 with those `clock_gettime` symbols around.

I'm generally a bit terrified of what's going to happen when people on El Cap switch over to Xcode 8 en mass, heh.